en/flight_controller/snapdragon_flight_dev_environment_installation.md

```diff
@@ -0,0 +1,39 @@
+# Snapdragon Development Environment Installation
+In order to get your snapdragon into the air, a few things need to be installed on your local PC.
```

Comment:
As in some previous note I can no longer find, might be worth adding a note here that firmware is not build by default and downloaded from QGC (which is why you need to build it yourself).

Reply:
It might also be good to make sure that this has everything relevant in the devguide about Snapdragon installation - so we could in theory just have this doc and remove the one in Devguide.

Reply:
This does indeed replace the instructions in the dev guide, they are no longer needed.

Added a note, not sure if this is the best place for it though

Reply:
@nicovanduijn Thanks very much - issue created to do this work in devguide https://github.com/PX4/Devguide/issues/521

Just to be very pedantic though, I will be assuming that all the material in Sysroot Installation, 
Update ADSP firmware, References can be deleted (see https://dev.px4.io/en/setup/dev_env_linux_ubuntu.html#snapdragon-flight )

Reply:
Thanks for pointing this out. I am not sure anymore what the minimal set of steps required is on a fresh install, I will double check. This is actually the point that triggered my initial frustration with the documentation:
The dev guide (https://dev.px4.io/en/setup/dev_env_linux_ubuntu.html#snapdragon-flight)  starts off by saying to follow the steps in cross_toolchain (https://github.com/ATLFlight/cross_toolchain), and then in the next section (sysroot installation) goes on to duplicate a subset of the steps of the tutorial previously linked to, but with slight differences. To top it all off, there's a link to another set of instructions, which contains a mixture of all the above (https://github.com/ATLFlight/cross_toolchain/blob/sdk3/README.md#sysroot-installation)

Once I'm 100% sure I have the minimal set of steps required on a fresh install, I would suggest to remove this maddness completely

Reply:
Ok, I double-checked what I thought I already knew. The dev guide is outdated, so is one of the cross_toolchain tutorials. The sysroot is installed along with the SDK. I added explicit instructions to append setting the environment variables to .bashrc, just to be clear. (The installer tells you to do so, but it isn't super obvious I think)

Bottom line: the dev guide can be deleted. The new instructions should contain the minimal required set of steps for a fresh install.

Reply:
Awesome! I will do so once this is merged (specifically I will remove the existing content and link to your docs in the user guide)